docs/pangeo_forge_recipes/development/release_notes.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## v0.8.4
+
+- Added `serialization` module along with `BaseRecipe.sha256` and `FilePattern.sha256` methods. Collectively,
+this provides for generation of deterministic hashes for both recipe and file pattern instances. Checking these
+hashes against those from a prior version of the recipe can be used to determine whether or not a particular recipe
+instance in a Python module (which may contain arbitrary numbers of recipe instances) has changed since the last time the instances in that module were executed. The file pattern hashes are based on blockchain built cumulatively from all of the index:filepath pairs yielded by the pattern's `self.items()` method. As such, in cases where a new pattern is intended to append to an existing dataset which was built from a prior version of that pattern, the pattern hash can be used to determine the index from which to begin appending. This is demonstrated in the tests. {pull}`349`
```

Comment:
Let's wrap this line better.

Reply:
👍 https://github.com/pangeo-forge/pangeo-forge-recipes/pull/349/commits/8303cfc0bd9d854d5dac5850c57375b4569e0946